app/plugins/logging.js

```diff
@@ -1,10 +1,7 @@
-const config = require('../config')
-
 module.exports = {
   plugin: require('hapi-pino'),
   options: {
     logPayload: true,
-    prettyPrint: config.isDev,
```

Comment:
What's the reason for removing this?

I can see this being used in the following repos:
- `ffc-pay-web`
- `ffc-pay-processing`
- `ffc-pay-request-editor`
- `azure-service-bus-message-hospital`

All of which bar the last have this configured.

Reply:
It no longer exists in the updated version of `hapi-pino`.

There is a separate package that needs to be installed and configured to get _similar_ capability, [`pino-pretty`](https://github.com/pinojs/pino-pretty)

The pretty only kicks in for local dev and we removed it from all the SFI services without issue.  Updated Pino logs a bit neater than older ones so our original need for it is gone so didn't see the value in adding another dependency.

Reply:
Agree -- it sounds like we should remove this from the other repos that use it too?

Can you create a tech debt ticket to do so?

Reply:
Agree -- it sounds like we should remove this from the other repos that use it too?

Can you create a tech debt ticket to do so?